client/src/components/Q&A/IndividualAnswer.jsx

```diff
@@ -29,9 +30,14 @@ const IndividualAnswer = ({ answer, handleHelpful, handleReport }) => {
         {showPhotos()}
       </p>
 
-      <div className="answer-helpfulness">
-        Helpful? <span onClick={() => handleHelpful(answer)}>Yes</span> ({helpfulness}) <button className="answer-report" onClick={(e) => handleReport(e, answer)}>Report</button>
-      </div>
+      <p className="answer-helpfulness">
+        Helpful? <span onClick={() => handleHelpful(answer)}>Yes</span> ({helpfulness}) {!report && <button className="answer-report"
+          onClick={() => {
+            setReport(true);
```

Comment:
I'm not familiar with the BRD here. Should it be possible for a user to change their answer? Or once they've submitted that it's helpful, should that be final?

Reply:
@joshgarza The BRD only states that clicking on the helpful will increment it by 1. I still need to implement a way for the increment to only inc. by 1 per user..